instrumentation/servlet/servlet-3.0/javaagent-unit-tests/src/test/java/io/opentelemetry/javaagent/instrumentation/servlet/v3_0/snippet/TestUtil.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.servlet.v3_0.snippet;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class TestUtil {
+
+  public static byte[] readFileBytes(String resourceName) throws IOException {
```

Comment:
Can be private?
```suggestion
  private static byte[] readFileBytes(String resourceName) throws IOException {
```

Reply:
readFileBytes is not only used for readFile function, but also for the outer test as a different way to read the input.
Change the two functions name to readFileAsBytes and readFileAsString to avoid further confusion.